src/components/translations_terms.ts

```diff
@@ -28,6 +28,7 @@ export const CfTerms = {
     [CommandResult.ValueCellIsInvalidFormula]: _t(
       "At least one of the provided values is an invalid formula"
     ),
+    [CommandResult.TargetOutOfSheet]: _t("The range cannot target out of the current sheet."),
```

Comment:
Message isn't very clear IMO. Since we can leave the side panel open while changing sheet, talking about current sheet is dubious. "The range" when there is no clear way to know about which range we are talking about is also not very clear.

"All the ranges should be in the same sheet as the conditional format" maybe ?

Reply:
Still valid, the message 
![image](https://github.com/user-attachments/assets/42ad2d50-1ea7-45e3-be70-994e6e5439cc)

does not seems very english/clear. Maybe it's just me. And we should probably put the range between quotes.